bower-publish.sh

```diff
@@ -15,13 +19,13 @@ rm -rf bower-chosen/*
 cp public/bower.json public/*.png public/chosen.jquery.js public/chosen.css bower-chosen/
 cd bower-chosen
 
-LATEST_VERSION=$(git diff bower.json | grep version | cut -d':' -f2 | cut -d'"' -f2 | tail -1)
+LATEST_VERSION=`git tag -l "*.*.*" | sed 's/^v//' | sort -V | tail -1`
 
-if [ -z $LATEST_VERSION ] ; then
+if [ "$LATEST_VERSION" = "$CHOSEN_VERSION" ] ; then
   echo "No Chosen version change. Skipped tagging"
 else
-  echo "Chosen version changed. Tagging version ${LATEST_VERSION}\n"
-  git tag -a "v${LATEST_VERSION}" -m "Version ${LATEST_VERSION}"
+  echo "Chosen version changed. Tagging version ${CHOSEN_VERSION}\n"
+  git tag -a "${CHOSEN_VERSION}" -m "Version ${CHOSEN_VERSION}"
```

Comment:
@koenpunt I think after your handful of pending changes get merged, we should release a new version.

Check out the draft release notes here: https://github.com/harvesthq/chosen/releases -- since at least #2440 and #2446 are both _not_ backwards compatible, I think semver dictates that we should jump to 2.0.0 for the next release.  What do you think?  (At the _very_ least, minor-version bump to 1.5.0.  This is not a teeny-version bump.)